### ISSUE TITLE
How to add `tail.writer` using `import`?

### ISSUE THREAD

**Issue by alexsobolenko** [pytesNET/tail.writer#11]
I want to use it into vue-component, for what I need to import lib.
I use this code:
```
import 'tail.writer/js/tail.writer.js';
...
document.addEventListener("DOMContentLoaded", function(){
    tail.writer("textarea", {
        ...
    });
});
```
... and get error in FF devtools:
```
TypeError: d is undefinedtail.writer.js:148
    js tail.writer.js:148
    __WEBPACK_AMD_DEFINE_RESULT__ tail.writer.js:13
    js tail.writer.js:13
    js tail.writer.js:36
```
What am I doing wrong?

**Comment by SamBrishes** [comment]
Hellow,

thanks for the bug report, should to be the same issue as in [#35](https://github.com/pytesNET/tail.DateTime/issues/35) on my tail.DateTime script. 

I'm using `this` instead of `window` on the IIFE factory function, which doesn't work on package bundler as WebPack or JS Frameworks as VueJS (for example). The 'this' solution seemed fine as I tested it with browserify, but it was a horrible idea.

In short: You did nothing wrong, I was the offender. I'll fix it as soon as possible.

Sincerely,
Sam.

**Comment by jasonbarry** [comment]
Hi @SamBrishes  – could this be repackaged so that we can import from the npm module without having to use a script tag / CDN? 

Using 

```js
import { writer } from 'tail.writer'
```

I get 

    undefined is not an object (evaluating 'd.body')

**Comment by last-partizan** [comment]
@SamBrishes looks like simple fix, can you do it? or i can try to make pull-request.
